src/coord/src/coord.rs

```diff
@@ -1391,8 +1410,15 @@ where
     ///
     /// TODO: In the future the coordinator should perhaps track a table's upper and
     /// since frontiers directly as it currently does for sources.
-    fn persisted_table_allow_compaction(&self, since_updates: &[(GlobalId, Antichain<Timestamp>)]) {
+    fn persisted_table_allow_compaction(
```

Comment:
is the todo above this line now stale?

Reply:
I think maybe not? But I also don't know exactly what the comment was aiming at. As it is, we still need this specialized code for tracking the sinces of the backing source, since they are not covered yet by the regular tracking logic. We still have the situation that tables sources always have an index stuck to them, and there can never be a bare "table source".